graphql/README.md

```diff
@@ -29,7 +29,17 @@ Entry point is `src/adapters/webserver/main.py`, which will start the applicatio
 - `graphql` is run via the `webserver` package.
 
 ## Deployment
-todo
+Compiled packages are stored at the [NASA Github packages page](https://github.com/orgs/nasa/packages/container/package/orca-graphql).
+Once a new version is ready for deployment, perform the following steps:
+1. Choose a new version number following [semantic versioning practices](https://semver.org/) and replace the existing value in the `graphql/Dockerfile`.
```

Comment:
We could make this as dynamic variable in Dockerfile and effectively skip this step.

Reply:
Additional research/implementation card required.

Reply:
Changed